.run/ORT Help.run.xml

```diff
@@ -0,0 +1,11 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="ORT Help" type="JetRunConfigurationType">
+    <option name="MAIN_CLASS_NAME" value="org.ossreviewtoolkit.cli.OrtMainKt" />
+    <module name="oss-review-toolkit.cli.main" />
+    <option name="PROGRAM_PARAMETERS" value="--help" />
+    <shortenClasspath name="ARGS_FILE" />
+    <method v="2">
+      <option name="Make" enabled="true" />
+    </method>
+  </configuration>
+</component>
```

Comment:
Missing new line

Reply:
These files are auto-generated by the IDE, and [in ORT Server we also committed them without the final newline](https://github.com/eclipse-apoapsis/ort-server/tree/main/.run). Do you believe I still should add a final newline?

Reply:
Oh sorry I approved, without checking first that auto merge was enabled.
Hope that's ok for you @nnobelis, because I think we should not manually touch these IDE generated files.